src/components/MovieForm/MovieForm.tsx

```diff
@@ -0,0 +1,152 @@
+import styles from "../MovieForm/MovieForm.module.css";
+import React from "react";
+import {MovieDetailsData} from "../../types/MovieDetailsData";
+
+interface MovieFormProps {
+    initialMovieInfo?: MovieDetailsData,
+    handleSubmit: (movieData: MovieDetailsData) => void;
+}
+
+const MovieForm: React.FC<MovieFormProps> = ({ initialMovieInfo, handleSubmit }) => {
+    return (
+        <form className={styles.movieForm}
+              onSubmit={(e) => {
+                  e.preventDefault();
+                  const form = e.target as HTMLFormElement;
+                  const formData = new FormData(form);
+                  const formDataObject = Object.fromEntries(formData) as Record<string, string>;
+
+                  const genres = formData.getAll("genres") as string[];
```

Comment:
`as string[];` is not super correct. Because you have some predefine genres and you can create like enum and setup here that it will be not any string but some enum genres